pkg/client/outofband/doc.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package outofband provides support for the Out-of-Band protocols:
+// https://github.com/hyperledger/aries-rfcs/blob/master/features/0434-outofband/README.md.
+//
+// Create your client:
+//
+// ctx := getFrameworkContext()
+// client, err := outofband.New(ctx)
+// if err != nil {
+//     panic(err)
+// }
+//
+// You can create requests and invitations with client.CreateRequest() and client.CreateInvitation()
+// respectively.
+//
+// You can accept out-of-band requests and invitations received via out of band channels.
+// If you have a request or an invitation on hand you can use the client.AcceptRequest() and
+// client.AcceptInvitation() respectively. These return the ID of the newly-created connection
+// record.
+//
```

Comment:
Do you have a code example of this scenario?

Reply:
https://github.com/hyperledger/aries-framework-go/blob/master/pkg/client/didexchange/example_test.go

Reply:
I'm working on adding example tests as per @rolsonquadras 's suggestion.

Reply:
But to answer your question: yes, in the out-of-band BDD tests, example: https://github.com/hyperledger/aries-framework-go/blob/master/test/bdd/pkg/outofband/outofband_sdk_steps.go#L158

Reply:
@rolsonquadras @troyronda please check now